packages/nouns-api/src/utils.ts

```diff
@@ -32,7 +32,8 @@ export const getTokenMetadata = async (tokenId: string): Promise<undefined | Tok
   );
   const svg = Buffer.from(data.image.substring(26), 'base64');
   const png = await sharp(svg).png().toBuffer();
-  const imageCID = await storage.storeBlob(png);
+  const blob = new Blob([png]);
```

Comment:
Not really sure why this started failing (complaining about types). This seemed to fix it though.